src/main/resources/application.properties

```diff
@@ -1 +1,9 @@
 
+#Global configuration
+spring.application.name=api
+
+spring.datasource.url=jdbc:mysql://localhost:3306/db_spring
+spring.datasource.username=root
+spring.datasource.password=GranMEMEH2002@
```

Comment:
Attention quand tu mets des mots de passes dans les fichiers commités, surtout sur des projets publics.
Ce serait top d'utiliser des fichier d'environnement : https://www.twilio.com/blog/working-with-environment-variables-in-java